.github/workflows/windows_ci.yml

```diff
@@ -99,9 +103,7 @@ jobs:
       - name: Test pure V math module
         run: ./v -exclude @vlib/math/*.c.v test vlib/math/
       - name: Self tests
-        run: |
-          ./v -cg cmd\tools\vtest-self.v
-          ./v test-self
+        run: ./v -cg cmd\tools\vtest-self.v && ./v test-self vlib
```

Comment:
Are you sure that && works in the default windows shell for that runner?
Imho isolate/avoid such changes in PRs that do other things.

Reply:
Yes it works, but can keep such changes into PR that does a general cleanup of workflows.